security.rst

```diff
@@ -2299,13 +2299,27 @@ the ``ROLE_SUPER_ADMIN`` permission:
     #[IsGranted('ROLE_ADMIN')]
     class AdminController extends AbstractController
     {
-        #[IsGranted('ROLE_SUPER_ADMIN')]
+        #[IsGranted('ROLE_SUPER_ADMIN', message: 'You are not allowed to access the admin dashboard.')]
```

Comment:
We should maybe mention `message` is optional ?

Reply:
Added a little comment above this line :+1: